src/core/common/permissions/isSiteModerationScoped.ts

```diff
@@ -1,4 +1,5 @@
 export interface UserModerationScopes {
+  scoped?: boolean | null;
```

Comment:
this is a good idea! As much as we **can** do truthiness on a possibly undefined array of siteID's... it's not great to deal with in reality (lol)

Reply:
🙏